network/peer-monitoring-service/client/src/peer_states/mod.rs

```diff
@@ -17,6 +21,9 @@ pub mod node_info;
 pub mod peer_state;
 mod request_tracker;
 
+/// The frequency at which we update the logs and metrics for the peer monitoring states
+const LOGS_AND_METRICS_FREQUENCY_SECS: u64 = 120; // 2 minutes
```

Comment:
Wondering if this needs to be so high - is it too expensive to collect? If not, may be keep it 30-60s so that we don't loose too much of granularity?